### ISSUE TITLE
Ubuntu message refresh

### ISSUE THREAD

**Issue by ruscot** [cabal-club/cabal-desktop#320]
Hi I'm new to cabal-desktop and I download the latest release cabal-desktop_6.0.8_amd64.deb for ubuntu. And when I use it with a friend we have to refresh manually to see the message we send. I don't know if it's normal or not ?

**Comment by cblgh** [comment]
@ruscot yes, sorry! there seems to have been a bug introduced in 6.0.8 (which was in itself trying to fix another message-related bug!)

try https://github.com/cabal-club/cabal-desktop/releases/tag/v6.0.7 and let us know what's up? :)

**Comment by ruscot** [comment]
No problem I just saw it and wanted to tell you : ) 
I have no problem with 6.0.7 version ! It's probably what you said before ! ; ) 
Good luck with your application !
